acquia/cli
# Fixes #641: Trigger custom scripts after acli pull commands

**Motivation**

Fixes #641

**Proposed changes**


After any ACLI command is run, ACLI will check `$(pwd)/composer.json` for a `post-acli-($command name)` script and execute it if present. E.g., `post-acli-db:pull`.

**Alternatives considered**


**Testing steps**


1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. `cd` into a Drupal application directory containing a composer.json.
4. Add an entry to the `scripts` key in the composer.json:
```
        "post-acli-ide-list": [
            "echo \"Hello world\""
        ]
```
5. Run `[path to acli]/bin/acli ide-list -vvv`
6. Observe:
```
Executing composer script `post-acli-ide-list` defined in `/Users/matt.grasmick/Sites/projects/amad9/composer.json`
> echo "Hello world"
Hello world
[notice] Command: 'composer' 'run-script' 'post-acli-ide-list' [Exit: 0]
```

**Merge requirements**
